From klaas.van.gend@mvista.com  Wed Oct 28 09:44:20 2009
From: Klaas van Gend <klaas.van.gend@mvista.com>
Date: Wed, 28 Oct 2009 02:01:07 +0100
Subject: Staging: comedi: comedi_fc: checkpatch.pl fixes
To: gregkh@suse.de
Cc: Klaas van Gend <klaas.van.gend@mvista.com>
Message-ID: <1256691668-5453-1-git-send-email-klaas.van.gend@mvista.com>


This patch fixes all warnings as issued by checkpatch.pl.

Signed-off-by: Klaas van Gend <klaas.van.gend@mvista.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/comedi_fc.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

--- a/drivers/staging/comedi/drivers/comedi_fc.c
+++ b/drivers/staging/comedi/drivers/comedi_fc.c
@@ -53,7 +53,7 @@ unsigned int cfc_write_array_to_buffer(s
 
 	retval = comedi_buf_write_alloc(async, num_bytes);
 	if (retval != num_bytes) {
-		printk("comedi: buffer overrun\n");
+		printk(KERN_WARNING "comedi: buffer overrun\n");
 		async->events |= COMEDI_CB_OVERFLOW;
 		return 0;
 	}
@@ -65,7 +65,6 @@ unsigned int cfc_write_array_to_buffer(s
 
 	return num_bytes;
 }
-
 EXPORT_SYMBOL(cfc_write_array_to_buffer);
 
 unsigned int cfc_read_array_from_buffer(struct comedi_subdevice *subd,
@@ -84,7 +83,6 @@ unsigned int cfc_read_array_from_buffer(
 
 	return num_bytes;
 }
-
 EXPORT_SYMBOL(cfc_read_array_from_buffer);
 
 unsigned int cfc_handle_events(struct comedi_device *dev,
@@ -102,7 +100,6 @@ unsigned int cfc_handle_events(struct co
 
 	return events;
 }
-
 EXPORT_SYMBOL(cfc_handle_events);
 
 MODULE_AUTHOR("Frank Mori Hess <fmhess@users.sourceforge.net>");
